logging/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "nodejs-docs-samples-logging",
+  "description": "Node.js samples for Google Cloud Logging.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "scripts": {
+    "list": "node list.js",
+    "write": "node write.js",
+    "export": "node export.js"
+  },
+  "dependencies": {
+    "gcloud": "^0.27.0"
+  }
```

Comment:
I think it's unnecessary, but if you guys want it, what version do you want me to specify? It's being tested on v0.10, v0.12, and v5.x